docker/kites/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM debian:latest
+MAINTAINER Simone Zaccariello <simone.zaccariello@gmail.com>
+# install binary and remove cache
+RUN apt-get update \
+    && apt-get install -y iperf3 netsniff-ng nano tcpdump screen \
+    && rm -rf /var/lib/apt/lists/*
+
+# Expose the default iperf3 server port
+EXPOSE 5201
+
+# entrypoint allows you to pass your arguments to the container at runtime
```

Comment:
We don't need these comments. There is the official docker documentation for that.